mantidproject/mantid
# Sample Transmission Calculator: remove option to change y-axis to logarithmic

**Description of work.**
In the Sample Transmission Calculator, the y-axis of the plot would disappear when it was changed from linear to logarithmic. This was because all of the y-axis values were originally less than 1 so the logs would be negative, and negative values are not shown on the plot. Therefore the option to change the Y-Axis has been removed.

**To test:**
1. Go to Interfaces -> General -> Sample Transmission Calculator.
2. Right-click the plot and check that there is no 'Y-Axis' option.

Fixes #24981 
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
